packages/block-library/src/footnotes/edit.js

```diff
@@ -12,8 +14,18 @@ export default function FootnotesEdit( { context: { postType, postId } } ) {
 		postId
 	);
 	const footnotes = meta?.footnotes ? JSON.parse( meta.footnotes ) : [];
+	const blockProps = useBlockProps();
+
+	if ( ! footnotes.length ) {
+		return (
+			<Placeholder { ...blockProps }>
+				{ __( 'No footnotes yet.' ) }
+			</Placeholder>
```

Comment:
I think the `Placeholder` component might need to be wrapped in a `div` element, otherwise the component's margin styles overrides a theme's layout styles, causing the placeholder to be rendered flush against other blocks. For example, here the placeholder state is right next to the adjacent paragraph block:

<img width="968" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/a9d759df-fc88-47e2-a271-17d9f8489a86">

As an example, the Table of Contents block has a wrapper `div` for its placeholder state, so it has a little more breathing room:

<img width="964" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/b1ee98c1-13df-4b90-87f5-3a5087ed5b63">